package/src/bin/deployment-example.toml

```diff
@@ -0,0 +1,44 @@
+# This manifest describes the servers that omicron will be installed to, along
+# with any ancillary information specific to a given server.
+#
+# It is ingested by the `thing-flinger` tool.
+
+# This must be an absolute path
+local_source = "/Users/ajs/oxide/omicron"
+
+[builder]
+# `server` must refer to one of the `servers` in the servers table
+server = "atrium"
+omicron_path = "/home/andrew/oxide/omicron"
+
+# Git branch, sha, etc...
+git_treeish = "thing-flinger2"
+
+[deployment]
+servers = ["sock", "buskin"]
+rack_secret_threshold = 2
+
+# Location where files to install will be placed before running
+# `omicron-package install`
+#
+# Since usernames may vary per server, this directory is relative to the user's
```

Comment:
Does that mean if I put in `/opt/oxide/staging/`  it'll be interpreted as `$HOME//opt/oxide/staging`?